lib/ddtrace.rb

```diff
@@ -29,10 +29,13 @@ def self.tracer
 if defined?(Rails::VERSION)
   if Rails::VERSION::MAJOR.to_i >= 3
     require 'ddtrace/contrib/rails/framework'
+    require 'ddtrace/contrib/rails/middlewares'
 
     module Datadog
       # Railtie class initializes
       class Railtie < Rails::Railtie
+        config.app_middleware.use(Datadog::Contrib::Rails::TraceMiddleware)
```

Comment:
It doesn't conflict with https://github.com/DataDog/dd-trace-rb/pull/170/files#diff-f9f184a4f121cd58a4162819f69200f7R57?